scripts/travis/build-all-in-one-image.sh

```diff
@@ -36,7 +36,7 @@ run_integration_test() {
 
 upload_to_docker() {
   # Only push the docker container to Docker Hub for master branch
-  if [[ ("$BRANCH" == "master" || $BRANCH =~ ^v[0-9]+\.[0-9]+\.[0-9]+$) && "$DOCKERHUB_LOGIN" == "true" ]]; then
+  if [[ ("$BRANCH" == "master" || $BRANCH =~ ^v[0-9]+\.[0-9]+\.[0-9]+$) && "${DOCKERHUB_LOGIN:-false}" == "true" ]]; then
```

Comment:
I'd say it's preferable to handle default values at the top of the script, similar to BRANCH and GOARCH, i.e. add
```
DOCKERHUB_LOGIN=${DOCKERHUB_LOGIN:-false}
```

Reply:
nice suggestion, updated the PR with the change.